RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -55,7 +57,8 @@
     originRadius = 0.02,
     nSigmaZ = 4.0
 ))
-trackingPhase1.toModify(lowPtTripletStepTrackingRegions, RegionPSet = dict(ptMin = 0.35)) # FIXME: Phase1PU70 value, let's see if we can lower it to Run2 value (0.2)
+trackingPhase1.toModify(lowPtTripletStepTrackingRegions, RegionPSet = dict(ptMin = 0.2))
```

Comment:
if blue is baseline, then it looks like lowpttriplet step time has reduced.
Was your test done with the ptMin = 0.2 ?